api/node/index.ts

```diff
@@ -76,6 +76,12 @@ export interface ImageData {
      *  Returns the height of the image in pixels.
      */
     get height(): number;
+
+    /**
+     * Returns the path of the image, if it was loaded from disk. Otherwise
+     * the property is undefined.
+     */
+    readonly path: string | undefined;
```

Comment:
is there not some syntax with `?` instead of `| undefined`?

Reply:
Yes indeed, I wonder why I didn't use that. Ahh, I had it in my buffer but the file wasn't saved when committing %-)